comp_loinc/build.py

```diff
@@ -1,19 +1,66 @@
-import typer
-from typing import List
-from ingest.part_ingest import PartOntology
-from ingest.code_ingest import CodeIngest
+"""CLI
+
+TODO's
+  1. Flesh out documentation for each param for each command.
+  2. Change args from positional to named, e.g.: https://typer.tiangolo.com/tutorial/parameter-types/number/
+    i. if i can't make them required, then i can raise an err if any are missing
+"""
 import os
 import subprocess
+from pathlib import Path
+
+import typer
+
+try:
+    from comp_loinc.ingest.part_ingest import PartOntology
+    from comp_loinc.ingest.code_ingest import CodeIngest
+except ModuleNotFoundError:
+    from ingest.part_ingest import PartOntology
+    from ingest.code_ingest import CodeIngest
 
-app = typer.Typer()
 
+app = typer.Typer(help='CompLOINC. A tool for creating an OWL version of LOINC.')
+# TODO: PR commment: Does he want this moved to a config? Or no defaults at all?
+PROJECT_DIR = os.path.join(os.path.dirname(__file__), '..')  # todo double check
+DEFAULTS = {
```

Comment:
Added some defaults. @putmantime Not sure if this is something you want though.

If you do like it, I could leave these here, or move them to a separate, `config.py` file.